spec/features/user_profiles_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+require 'helpers'
+
+feature 'user can view index of users' do
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'users can view index of users' do
+    sign_in_as(user)
+    visit users_path
+    within(".column") do
+      expect(page).to have_content(user.username)
+    end
+  end
+
+  scenario 'visitors can view index of users' do
+    sign_in_as(user)
+    visit users_path
+    within(".column") do
+      expect(page).to have_content(user.username)
+    end
+  end
+end
+
+feature 'user can view profile' do
+  let(:review) { FactoryGirl.create(:review, user: user) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'users can view their profile' do
```

Comment:
If you want to write the test below name this test as `scenario 'user can navigate to their own profile from the users index page'`.